File: src/main/java/com/extendedae_plus/common/registry/block/assemblerMatrix/coreAdvancedSpeed/BlockAdvancedSpeed.java
package com.extendedae_plus.common.registry.block.assemblerMatrix.coreAdvancedSpeed;

import com.extendedae_plus.common.init.ModItems;
import com.glodblock.github.extendedae.common.blocks.matrix.BlockAssemblerMatrixBase;
import net.minecraft.world.item.Item;

public class BlockAdvancedSpeed extends BlockAssemblerMatrixBase<BlockEntityAdvancedSpeed> {
    @Override
    public Item getPresentItem() {
        return ModItems.CoreAdvancedSpeed.get();
    }
}

File: src/main/kotlin/com/extendedae_plus/common/registry/block/assemblerMatrix/coreAdvancedSpeed/BlockAdvancedSpeed.kt
package com.extendedae_plus.common.registry.block.assemblerMatrix.coreAdvancedSpeed

import com.extendedae_plus.common.init.EAEPItems
import com.fish.fishlib.util.extension.invoke
import com.glodblock.github.extendedae.common.blocks.matrix.BlockAssemblerMatrixBase
import net.minecraft.world.item.BlockItem

class BlockAdvancedSpeed : BlockAssemblerMatrixBase<TileAdvancedSpeed>() {
    override fun getPresentItem(): BlockItem = EAEPItems.CoreAdvancedSpeed()
}
